specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/virtualNetworkGateway.json

```diff
@@ -2459,6 +2459,24 @@
       ],
       "description": "VPN client revoked certificate of virtual network gateway."
     },
+    "RadiusServer": {
```

Comment:
>"RadiusServer" [](start = 4, length = 14)

please add at least one example that contains radius servers to all APIs that are affected by this change

Reply:
Done.